docs/package.json

```diff
@@ -6,7 +6,7 @@
     "buildDependencies": "yarn workspace @stellar/lyra-api build",
     "docusaurus": "docusaurus",
     "start": "yarn buildDependencies && docusaurus start",
-    "build": "yarn buildDependencies && docusaurus build",
+    "build": "docusaurus build",
```

Comment:
docusaurus can't build unless lyra-api has been built. try removing @stellar/lyra-api/build and run this. it'll error

Reply:
The netlify build tho runs `yarn workspace @stellar/lyra-api build && yarn workspace docs build` so it will always already have lyra-api recently built

Reply:
yeah, i'd say running this locally for testing purposes should still work without knowing to have to build lyra-api though. 

the Netlify command should actually just be changed to `yarn workspace docs build`

Reply:
Chatted about it, I've got another branch in progress, gonna hash out build command changes there